src/character.cpp

```diff
@@ -8416,15 +8415,15 @@ int Character::height() const
 {
     switch( get_size() ) {
         case creature_size::tiny:
-            return init_height - 100;
+            return init_height * 0.5;
```

Comment:
```suggestion
            return init_height * 0.4;
```
See my earlier comment.
These should be defined in JSON rather than in c++ really, but that's for later.

Reply:
Yeah, I also thought about JSONizing it. Maybe I'll get to it after I finish this PR.